docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
   - Click **Continue**.
 - Click the **Save** button and wait for Strapi to restart.
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
+
+- Click the **"+ Add Content Type"** button.
```

Comment:
Since the modal is already opened this line is not needed.

Reply:
This line is needed still as the modal is not really open yet.  See the previous comment.  You have designed multiple ways to do something that are evident after a little use, but as I said earlier, the left hand menu is already familiar.